tracks/elm/exercises/bob/mentoring.md

```diff
@@ -0,0 +1,41 @@
+This is a good exercise to talk about bringing domain concepts (such as `isYelling`, `isQuestion` and so on) in to the code, as it is quite easy and most students will do it automatically.
+
+This exercise introduces some string handling functions, such as `String.trim`, `String.endsWith`, `String.any`, `String.isEmpty`, `Char.isAlpa` and `Char.isUpper`
+
+There are some good and sensible opportunities to use point free style.
+
+There are also some general code quality things to discuss, such as keep related logic together, and ordering the code by importance. Students should strive to communicate what the code does in as few lines as possible (preferably just the main function and argument names). They show also strive to communicate how it does it in as few lines as possible, preferably just the main function body.
```

Comment:
```suggestion
There are also some general code quality things to discuss, such as keep related logic together, and ordering the code by importance. Students should strive to communicate what the code does in as few lines as possible (preferably just the main function and argument names). They should also strive to communicate how it does it in as few lines as possible, preferably just the main function body.
```

Reply:
I have removed the comma here, which I think is what you wanted

Reply:
Hi, sorry for the late feedback.

No, if you scroll the "Suggested change" box, the diff is `show` -> `should`.